src/ColorPicker/Models/MathHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ColorPicker.Models
+{
+    internal static class MathHelper
+
+    {
+        public static double Clamp(double value, double min, double max)
```

Comment:
Why not use `Math.Clamp`?

Reply:
There is no Math.Clamp in .NET Framework sadly